src/controllers/dlgprefcontroller.cpp

```diff
@@ -259,8 +258,10 @@ void DlgPrefController::enumeratePresets() {
         return;
     }
 
-    const QList<PresetInfo> presets = pie->getPresetsByExtension(
+    //Making the list of presets in the alphabetical order
```

Comment:
Please add space between // and comment. without space is for code only //code
https://www.mixxx.org/wiki/doku.php/coding_guidelines#comments